packages/fileeditor/src/searchprovider.ts

```diff
@@ -59,6 +60,15 @@ export class FileEditorSearchProvider
     return this.widget.content.model;
   }
 
+  async startQuery(
+    query: RegExp,
+    filters: IFilters | undefined
+  ): Promise<void> {
+    const promise = await super.startQuery(query, filters);
+    await this.highlightNext(false, true);
+    return promise;
```

Comment:
I guess we could simplify this:

```suggestion
    await super.startQuery(query, filters);
    await this.highlightNext(false, true);
```